boards/arm/xilinx_kv260_r5/doc/index.rst

```diff
@@ -0,0 +1,108 @@
+.. _xilinx_kv260_starter_kit_r5:
+
+Xilinx KV260 Development Board RPU Cortex-R5
+############################################
+
+Overview
+********
+This configuration provides support for the RPU, real-time processing unit on Xilinx
+KV260 development board, it can operate as following:
+
```

Comment:
Add link to a good page for kv260 info in here somewhere.
Match what is done for other boards.

Reply:
As @microbuilder pointed out on our call.  The page should also have a picture of the board to match what is normally done for other boards.